hudi-io/src/test/java/org/apache/hudi/io/hfile/TestHFileReader.java

```diff
@@ -227,11 +351,15 @@ static Stream<Arguments> testArgsReadHFilePointAndPrefixLookup() {
         Arguments.of(
             "/hfile/hudi_1_0_hbase_2_4_9_64KB_NONE_5000.hfile",
             5000,
+            KEY_CREATOR,
             Arrays.asList(
                 // before first key
-                new KeyLookUpInfo("", SEEK_TO_BEFORE_FIRST_KEY, "", ""),
-                new KeyLookUpInfo("a", SEEK_TO_BEFORE_FIRST_KEY, "", ""),
-                new KeyLookUpInfo("hudi-key-0000000", SEEK_TO_BEFORE_FIRST_KEY, "", ""),
+                // before first key
+                new KeyLookUpInfo("", SEEK_TO_BEFORE_FILE_FIRST_KEY, "hudi-key-000000000", "hudi-value-000000000"),
+                new KeyLookUpInfo("as", SEEK_TO_BEFORE_FILE_FIRST_KEY, "hudi-key-000000000", "hudi-value-000000000"),
+                // backward seekTo before first key is allowed and safe
```

Comment:
is there a typo here. by safe you mean, we just expect SEEK_TO_BEFORE_FILE_FIRST_KEY as the return status?

Reply:
Yes